src/commands/deactivate-plugin.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Deactivate Plugin
+ *
+ * @param slug - Plugin name. If empty string given, the current plugin name
+ *               will be retrieved from CYPRESS_WP_CURRENT_PLUGIN environment
+ *               variable
+ *
+ * @example
+ * Deactivate currently tested plugin
+ * ```
+ * cy.deactivatePlugin()
+ * ```
+ *
+ * @example
+ * Deactivate Classic Editor
+ * ```
+ * cy.deactivatePlugin('classic-editor')
+ * ```
+ */
+export const deactivatePlugin = (slug = ''): void => {
+  cy.visit('/wp-admin/plugins.php');
+  if ('' === slug) {
+    // Try to retrieve current plugin from ENV.
+    cy.getCurrentPlugin().then(current => {
+      if ('string' === typeof current && '' !== current) {
+        cy.get('body').then($body => {
+          // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+          if ($body.find(`#deactivate-${current}`).length > 0) {
+            // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+            cy.get(`#deactivate-${current}`).click();
```

Comment:
@cadic just wanted to add a note here that `#deactivate-PLUGIN_SLUG` is added in WP 5.5 (https://core.trac.wordpress.org/ticket/49916). So, this may not work in WP version < 5.5. especially in our minimum env test.

Reply:
@iamdharmesh thank you for this notice!